.github/workflows/latest_tags.yml

```
@@ -0,0 +1,116 @@
+        # mainnet for instance.
+        options:
+          - arabica
+          - mocha
+
+jobs:
+  # This job pulls the latest tag information for the repos listed in the matrix
+  # strategy
+  logLatestRelease:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # If we want more repos, we can just add them here.
+        repo: [celestia-app, celestia-core, celestia-node]
+    env:
+      # This workflow is set for the celestiaorg github org, if we need other
+      # orgs we will need to restructure this workflow
+      owner: celestiaorg
+    # Define the outputs for other steps to access
+    outputs:
+      celestia-app_latest_tag: ${{ steps.set_outputs.outputs.celestia-app_latest_tag }}
+      celestia-app_latest_sha: ${{ steps.set_outputs.outputs.celestia-app_latest_sha }}
+      celestia-core_latest_tag: ${{ steps.set_outputs.outputs.celestia-core_latest_tag }}
+      celestia-core_latest_sha: ${{ steps.set_outputs.outputs.celestia-core_latest_sha }}
+      celestia-node_latest_tag: ${{ steps.set_outputs.outputs.celestia-node_latest_tag }}
+      celestia-node_latest_sha: ${{ steps.set_outputs.outputs.celestia-node_latest_sha }}
+    steps:
+      - name: Log target owner/repo
+        run: "echo Pulling info from: '${{ env.owner }}/${{ matrix.repo }}'"
+
+      - name: Query /releases/latest endpoint
+        uses: octokit/request-action@v2.x
+        id: get_latest_release
+        with:
+          route: GET /repos/{owner}/{repo}/releases/latest
+          owner: ${{ env.owner }}
+          repo: ${{ matrix.repo }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Log latest tag
+        run: "echo latest tag: '${{ fromJSON(steps.get_latest_release.outputs.data).tag_name }}'"
+
+      - name: Query /git/ref/tag endpoint
+        uses: octokit/request-action@v2.x
+        id: get_latest_commit
+        with:
+          route: GET /repos/{owner}/{repo}/git/ref/tags/${{ fromJSON(steps.get_latest_release.outputs.data).tag_name }}
+          owner: ${{ env.owner }}
+          repo: ${{ matrix.repo }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Log latest sha
+        run: "echo latest commit: '${{ fromJSON(steps.get_latest_commit.outputs.data).object.sha }}'"
+
+      - name: Set outputs
+        id: set_outputs
+        run: |
+          echo "${{matrix.repo}}_latest_tag=${{ fromJSON(steps.get_latest_release.outputs.data).tag_name }}" >> $GITHUB_OUTPUT
+          echo "${{matrix.repo}}_latest_sha=${{ fromJSON(steps.get_latest_commit.outputs.data).object.sha }}" >> $GITHUB_OUTPUT
+
+  # This job creates a PR if there were changes. The changes are created when
+  # the workflow writes to the version files. This means that the workflow
+  # should be able to be run and will just be a no-op if the is no new release.
+  # Either way, since a PR is created, this projects breaking the docs on main.
```

```suggestion
  # Either way, since a PR is created, this projects won't break the docs on main.
```

I think this was a typo from me.

---

that I didn't fix* 😄

---

✔️ e7f37c6568b76030125b22b92fb025fdb89cec06